tests/e2e/prevent_duplicate_cluster_ids.go

```diff
@@ -38,7 +38,8 @@ import (
 	admissionapi "k8s.io/pod-security-admission/api"
 )
 
-var _ = ginkgo.Describe("Prevent duplicate cluster ID", func() {
+var _ = ginkgo.Describe("Prevent duplicate cluster ID", ginkgo.Label(p1, vanilla, block, file,
```

Comment:
Not all testcases in this file are p1. Please assign the following testcases as P0 :

1. Generate unique cluster id through configmap and create workloads
2. Generate cluster id and then set cluster id in vsphere config secret
		and remove cluster id field in vsphere config secret
3. Modify unique cluster id value in Configmap		
		
Rest all others can be assigned as P1